ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueCommitting.scala

```diff
@@ -276,20 +274,4 @@ object KeyValueCommitting {
     }
     outputs.result()
   }
-
-  private def contractKeyToStateKey(
-      key: GlobalKey
-  ): DamlStateKey = {
-    // NOTE(JM): The deserialization of the values is meant to be temporary. With the removal of relative
```

Comment:
This comment is no longer valid (see https://github.com/digital-asset/daml/pull/8519/files#diff-e355a2d6edc45d41fc83bf220e75220dc4d0873cd0d2641e9378eabb5535c368R260)